front/lib/api/assistant/citations.ts

```diff
@@ -35,10 +36,13 @@ export const getRefs = () => {
 /**
  * Prompt to remind agents how to cite documents or web pages.
  */
-export function citationMetaPrompt() {
+export function citationMetaPrompt(isUsingRunAgent: boolean) {
   return (
     "## CITING DOCUMENTS\n" +
     "You MUST cite ALL information retrieved from documents or web pages. Citations are critical for transparency and verification.\n" +
+    (isUsingRunAgent
+      ? "If you use information from a run_agent that is related to a document or a web page, you MUST include the citation markers exactly as they appear."
```

Comment:
is `run_agent` clear in this context ? Is the tool always named like this ?

if so I would use a constant somewhere and say:
If you use information from a sub agent via a "$TOOL_NAME" tool, ...


Just to make sure it's extra clear.

Reply:
Merging as is cause non blocker but I can happily add a commit if you have a smart idea 🙏🏻